manager/pom.xml

```diff
@@ -156,11 +156,6 @@
       <artifactId>logger-java-log4j</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-jacoco</artifactId>
```

Comment:
Is this an unused dependency?

Reply:
I check the `quarkus-jacoco` documentation, it mentioned that developer could use either plugin or dependency.
https://quarkus.io/guides/tests-with-coverage

`jacoco-maven-plugin` plugin is included with `measure-code-coverage` profile.